README.md

```diff
@@ -0,0 +1,49 @@
+# RIT-SPEX/latex-utils
```

Comment:
Maybe add a section for pre-requisite software.  At this point it should only be Python 3.X.